src/main/java/com/hyunec/cosmicbaseballinit/service/BattingService.java

```diff
@@ -0,0 +1,33 @@
+package com.hyunec.cosmicbaseballinit.service;
+
+import static com.hyunec.cosmicbaseballinit.domain.BattingResult.BALL;
+import static com.hyunec.cosmicbaseballinit.domain.BattingResult.HIT;
+import static com.hyunec.cosmicbaseballinit.domain.BattingResult.STRIKE;
+
+import com.hyunec.cosmicbaseballinit.domain.BattingResult;
+import java.util.Random;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class BattingService {
+
+	private static final Random RANDOM_GENERATOR = new Random();
+
+	public BattingResult batting() {
+		return getResult();
+	}
+
+	private BattingResult getResult() {
+
+		switch (RANDOM_GENERATOR.nextInt(BattingResult.values().length)) {
+			case 0:
+				return STRIKE;
+			case 1:
+				return HIT;
+			default:
+				return BALL;
+		}
+	}
+}
```

Comment:
```suggestion
@Service
public class BattingService {

    private static final Random RANDOM = new Random();

    public BattingResult batting() {
        return BattingResult.values()[RANDOM.nextInt(BattingResult.values().length)];
    }
}
```

`nit` 현재 요건을 위해서 아마 저라면 이렇게 할 것 같습니다.
향후 `BattingResult 를 생성해야 되는 전략이 변경된다면` strategy 로 분리할 듯 합니다.